src/Microsoft.CSharp/tests/CSharpArgumentInfoTests.cs

```diff
@@ -17,7 +17,10 @@ public class CSharpArgumentInfoTests
 
         private static readonly string[] Names =
         {
-            "arg", "ARG", "Arg", "Argument name that isn’t a valid C♯ name 👿🤢",
+            "arg", "ARG", "Arg", 
+#if !uap
```

Comment:
I think I would perfer we just disable this test for UAP with and active issue until we get it fixed.